src/backend/catalog/storage.c

```diff
@@ -198,6 +198,12 @@ void
 RelationDropStorage(Relation rel)
 {
 	PendingRelDelete *pending;
+	SMgrRelation srel;
+
+	srel = smgropen(rel->rd_node, rel->rd_backend == TempRelBackendId ?
```

Comment:
use rel->rd_backend directly? smgropen(rel->rd_node, rel->rd_backend, .....)

Reply:
OK，that's right